docs/ingressroute.md

```diff
@@ -528,7 +556,7 @@ A key feature of the IngressRoute specification is route delegation which follow
 > Any nameserver can hold a record for `app.heptio.com`, but without the linkage from the parent `heptio.com` nameserver, its information is unreachable and non authoritative.
 
 The "root" IngressRoute is the only entry point for an ingress virtual host and is used as the top level configuration of a cluster's ingress resources.
-Each root IngressRoute defines a `virtualhost` key, which describes properties such as the fully qualified name of the virtual host, any aliases of the virtual host (for example, a www. prefix), TLS configuration, etc.
+Each root IngressRoute defines a `virtualhost` key, which describes properties such as the fully qualified name of the virtual host (for example, a www. prefix), TLS configuration, etc.
```

Comment:
I think the "(for example, a www. prefix)" parenthesis should also go away with the removal of aliases